tests/php/Publisher/FilesystemPublisherTest.php

```diff
@@ -285,7 +285,7 @@ public function testRedirectorPageWhenPHP(): void
         $redirectorPage = RedirectorPage::create();
         $redirectorPage->URLSegment = 'somewhere-else';
         $redirectorPage->RedirectionType = 'External';
-        $redirectorPage->ExternalURL = 'silverstripe.org';
+        $redirectorPage->ExternalURL = 'https://example.com/path';
```

Comment:
This test is now inconsistent with `testRedirectorPageWhenHTMLOnly()`. Please update the value in that test as well.

Can you please also confirm that the only problem was the lack of `http(s)://` at the start of the URL? i.e. would `https://silverstripe.org` have worked? If so, this change is fine.

Reply:
> Can you please also confirm that the only problem was the lack of http(s):// at the start of the URL?

Yes it was the lack of a scheme, also works with http://

Have changed to https://example.org/path